README.md

```diff
@@ -1 +1,19 @@
-# blog_samples
\ No newline at end of file
+# Kenna Security Blog Samples
+
+These scripts accompany the Kenna blogs. 
+
+The [Kenna Security REST API](https://apidocs.kennasecurity.com/reference) samples provided are in
+different languages.  Each language will have its own development environment and different samples.
+For example Python could have assets and vulnerabilities samples, whereas PowerShell has assets and
+asset group samples.
+
+For more samples, please look at our [All_Samples](https://github.com/KennaSecurity/All_Samples)
+repository.  There you will find many samples, mostly in Ruby.
+
+For additional information about how you can use the Kenna API (employed in the sample scripts)
+to enhance/supplement your Kenna implementation, contact
+[Kenna Customer Success](mailto:customersuccess@kennascurity.com).
+
+All the code samples in this GitHub repository are offered “as is” and include no warranty of any kind.
+Use them at your own risk. In no event will Kenna be liable to end user or any other party for damages
+of any kind arising from the use of these samples.
```

Comment:
Good call-out here.  Can you copy the same MIT License that we have on toolkit?

KennaSecurity/toolkit@5bd1af20d9faeea03360a05672503afead7b13f7

Reply:
Added LICENSE.md.